packages/webapp/src/views/developer/file-upload-check.html

```diff
@@ -0,0 +1,52 @@
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
```

Comment:
Is this file actually in use? The check-metric-file.html view uses the common/file-upload-check.html partial. I think it may need deleting.